src/app/components/PriceChart.tsx

```diff
@@ -0,0 +1,110 @@
+  );
+
+  const currencyCode = useAppSelector(
+    (state) => state.currency.currentCurrency.code
+  );
+  const currencySimbol = useAppSelector(
+    (state) => state.currency.currentCurrency.symbol
+  );
+
+  const [currentPriceCoin, setCurrentPriceCoin] = useState("");
+  const [currentDate, setCurrentDate] = useState("");
+
+  useEffect(() => {
+    dispatch(
+      getCoinDataGraph({
+        currency: currencyCode,
+        days: "3",
+        coinId: "bitcoin",
+      })
+    );
+  }, [dispatch, currencyCode]);
+
+  useEffect(() => {
+    if (selectedCoins.length > 0) {
+      const lastDataPoint =
+        selectedCoins[0].prices[selectedCoins[0].prices.length - 1];
+      const lastPrice = lastDataPoint[1];
+      const lastDate = new Date(lastDataPoint[0]).toLocaleDateString();
+
+      setCurrentPriceCoin(`${lastPrice.toFixed(2)}`);
+      setCurrentDate(lastDate);
+    }
+  }, [selectedCoins, currencyCode]);
+
+  if (loading === "pending") {
+    return <div>Loading...</div>;
+  }
+
+  if (hasError) {
+    return <div>Error fetching data</div>;
+  }
+
+  const coin = selectedCoins.length > 0 ? selectedCoins[0] : null;
+
+  const formattedData = coin
+    ? coin.prices.map(([time, price]) => ({
+        date: new Date(time).toLocaleDateString(),
+        price,
+      }))
+    : [];
+
+  const handleMouseMove = (e: any) => {
+    if (e.activePayload && e.activePayload.length > 0) {
+      const { payload } = e.activePayload[0];
+      setCurrentPriceCoin(`${payload.price.toFixed(2)}`);
+      setCurrentDate(payload.date);
+    }
+  };
+
+  return (
+    <div className="w-1/2 h-[404px] flex flex-col dark:bg-[#191932] bg-white rounded-xl p-6">
+      <div className="flex flex-col">
+        <span className="text-xl font-normal dark:text-[#D1D1D1] text-[#191932] leading-6">
+          {coin
+            ? coin.id.charAt(0).toUpperCase() + coin.id.slice(1)
+            : "Loading..."}
+        </span>
+        <span className="text-[28px] font-bold leading-7 mt-6">
+          {currentPriceCoin
+            ? `${currencySimbol}${currentPriceCoin}`
+            : "Fetching..."}
+        </span>
+        <span className="text-base font-normal dark:text-[#B9B9BA] text-[#424286] leading-6 mt-4">
+          {currentDate}
+        </span>
+      </div>
+      <ResponsiveContainer width="100%" height={266}>
+        <AreaChart data={formattedData} onMouseMove={handleMouseMove}>
+          <defs>
+            <linearGradient id="color" x1="0" y1="0" x2="0" y2="1">
+              <stop offset="1%" stopColor="#7474F2" stopOpacity={0.6} />
+              <stop offset="60%" stopColor="#7474F2" stopOpacity={0.1} />
+            </linearGradient>
+          </defs>
+          <Area
+            dataKey="price"
+            stroke="#7878FA"
+            strokeWidth="3"
+            fill="url(#color)"
+          />
+
+          <XAxis dataKey="date" axisLine={false} tickLine={false} />
+
+          <Tooltip content={<></>} />
```

Comment:
Why does Tooltip don't have any content?

Reply:
I don't want to display anything in Tooltip because I'm displaying updated price outside chart. When I'm hovering over chart it is updating price number outside chart. Tooltip is used to display information inside chart as a box which is displayed when you hover over chart. If I delete content={<></>} inside tooltip i will have the same information presented twice, price will be displayed as a box inside chart and outside chart because I specified to have a price outside chart with the name of coin and date.